src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -431,6 +431,18 @@ export class SbbFormField implements ComponentInterface {
     await this._checkAndUpdateInputEmpty();
   }
 
+  /** Manually clears the input value. */
+  @Method() public async clear(): Promise<void> {
+    if (this._input.tagName !== 'INPUT') {
+      return;
+    }
+    (this._input as { value }).value = '';
+    toggleDatasetEntry(this._element, 'inputEmpty', this._isInputValueEmpty());
```

Comment:
to be consistent use
```suggestion
    await this._checkAndUpdateInputEmpty();
```